include/urcu/rculfhash.h

```diff
@@ -180,6 +205,52 @@ struct cds_lfht *cds_lfht_new_flavor(unsigned long init_size,
 			flags, NULL, flavor, attr);
 }
 
+/*
+ * cds_lfht_new_with_flavor_alloc - allocate a hash table tied to a RCU flavor.
+ * @init_size: number of buckets to allocate initially. Must be power of two.
+ * @min_nr_alloc_buckets: the minimum number of allocated buckets.
+ *                        (must be power of two)
+ * @max_nr_buckets: the maximum number of hash table buckets allowed.
+ *                  (must be power of two, 0 is accepted, means
+ *                  "infinite")
+ * @flavor: flavor of liburcu to use to synchronize the hash table
+ * @alloc: Custom memory allocator for hash table memory management.
+ * 		   NULL for default. If a custom allocator is used, then
```

Comment:
please align text with spaces (not tabs) with the first line ("Custom....").

Reply:
Fixed the spaces.

Reply:
Here I means that all tabs (within comments) should be changed into spaces. There is still a tab in there.